src/d3a/setup/area_events/cloud_coverage_event.py

```diff
@@ -30,15 +30,15 @@ def get_setup(config):
                 'House 1',
                 event_list=[ConfigEvents(12, {'cloud_coverage': 1})],
                 children=[
-                    Area('H1 PV', strategy=PVPredefinedStrategy(),
+                    Area('H1 PV', strategy=PVPredefinedStrategy(capacity_kW=0.25),
```

Comment:
Nit: if you happen to refactor, you could change from `'` to `"`. Same applies to the rest of the file/PR.